commands/log.go

```diff
@@ -32,3 +46,69 @@ Outputs event log messages (not other log messages) as they are generated.
 		return re.Emit(r)
 	},
 }
+
+var logLevelCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Change the logging level.",
+		ShortDescription: `
+Change the verbosity of one or all subsystems log output. This does not affect
+the event log.
+`,
+	},
+
+	Arguments: []cmdkit.Argument{
+		// TODO use a different keyword for 'all' because all can theoretically
```

Comment:
my suggestion is remove this todo. if it does clash then someone can fix it. or if you want give the level first and then an optional subsystem second on the command line. if they leave the subsystem off then treat that as all. we definitely definitely should not try to use * on the command line.

Reply:
I like the idea of having subsystem be optional, will go with that